config/public-api-v1.yml

```diff
@@ -31,31 +31,441 @@ paths:
+          format: markdown
+          description: "Further details about the fees for this course, if applicable."
+          example: fee_details
+        fee_international:
+          type: integer
+          description: "Fee for international students (optional)."
+          example: 13000
+        fee_domestic:
+          type: integer
+          description: "Fee for UK and EU students."
+          example: 9200
+        financial_support:
+          type: string
+          format: markdown
+          description: "Details about financial support offered, if any."
+          example: financial_support
+        findable:
+          type: boolean
+          description: |
+            Is this course currently visible on the Find Postgraduate Teacher Training service.
+          example: true
+        funding_type:
+          type: string
+          description: |
+            The type of funding that maybe provided to candidates, if any.
+          example: apprenticeship
+          enum:
+            - ""
+            - salary
+            - apprenticeship
+            - fee
+        gcse_subjects_required:
+          type: string
+          example: [ maths, english ]
+          enum:
+            - [maths, english]
+            - [maths, english, science]
+          description: "GSCE standard equivalent required for this level of course."
+        has_bursary:
+          type: boolean
+          description: "Are any bursaries available for this course?"
+          example: true
+        has_early_career_payments:
+          type: boolean
+          description: "Are early career payments available for this course?"
+          example: true
+        has_scholarship:
+          type: boolean
+          description: "Are scholarships available for this course?"
+          example: true
+        has_vacancies:
+          type: boolean
+          description: "Do any of the locations for this course have vacancies?"
+          example: true
+        how_school_placements_work:
+          type: string
+          format: markdown
+          description: |
+            Additional information about the schools applicants will be teaching in.
+          example: how_school_placements_work
+        interview_process:
+          type: string
+          format: markdown
+          description: |
+            Additional information about how the interview process will work for applicants.
+          example: interview_process
+        is_send:
+          type: boolean
+          description: "Does this course have a SEND specialism?"
+          example: true
+        last_published_at:
+          type: string
+          format: date-time
+          description: |
+            Timestamp of when changes to this course's additional information
+            sections was published last.
+          example: "2019-06-13T10:44:31Z"
+        level:
+          type: string
+          description: "The level of pupils this course is designed for."
```

Comment:
Reads kinda wrong...

The level of the course that it is designed for?

Reply:
Will ask PaulG to input.